scripts/cms-any/dispatch-ci.php

```diff
@@ -39,7 +39,14 @@
 
 $dispatchCiPath = '.github/workflows/dispatch-ci.yml';
 $ciPath = '.github/workflows/ci.yml';
+// old logic
 $shouldHaveDispatchCi = (is_module() || is_composer_plugin()) && !is_docs() && !is_gha_repository();
+// new logic to ensure that behat-extension has dispatch-ci.yml
+// We could possibly delete the old logic, though I'm halfway through updating existing PRs
+// so do not want to risk breaking this for the next run
+if (!has_wildcard_major_version_mapping()) {
+  $shouldHaveDispatchCi = true;
+}
```

Comment:
Why are we keeping two sets of logic? I don't want to merge that - especially with a comment saying "I'm halfway through"... that comment will get instantly stale.

Reply:
Either delete the comments and add a single comment saying why `has_wildcard_major_version_mapping` as false means `shouldHaveDispatchCi` should be true - or update the logic entirely with a comment saying why the logic is what it is.

Reply:
Updated